test/api_getProduct_test.dart

```diff
@@ -1858,4 +1858,57 @@ void main() {
     expect(result.product?.packagingTags, ['de:in-einer-plastikflasche']);
     expect(result.product?.quantity, '5.5 Liter');
   });
+
+  group('no nutrition data', () {
+    test('Null value', () async {
+      const String barcode = '4260633610236';
+
+      final ProductQueryConfiguration configurations =
+          ProductQueryConfiguration(
+        barcode,
+        fields: [ProductField.NO_NUTRITION_DATA],
+      );
+
+      final ProductResult result = await OpenFoodAPIClient.getProduct(
+        configurations,
+        user: TestConstants.TEST_USER,
+      );
+
+      expect(result.product?.noNutritionData, isFalse);
+    });
+
+    test('Empty value', () async {
+      const String barcode = '5449000000996';
```

Comment:
Those products are likely not to exist when the .net database gets refreshed.

I recommend that any test that expects a specific value first writes that value.

Reply:
Ok @stephanegigandet 

Just to confirm if I have well understood: 
- First, I create a product locally with the expected value
- Upload this product
- Fetch this product for the test?

Reply:
Yes, I think we have to do that so that tests can pass regardless of the initial state of the OFF test database.